src/components/Text/utils/Word.tsx

```diff
@@ -0,0 +1,73 @@
+import { FormattedGreekWord} from '../../../types';
+import { useRef } from 'react';
+
+
+interface WordProps {
+    onPhraseClick: (greekWords : FormattedGreekWord[]) => void;
+    englishWords : string,
+    isPhrase? : boolean,
+    containsSubWords? : boolean,
+    greekWords? : FormattedGreekWord[],
+    handleClick: (params: any) => any,
+  }
+  
+  
+  function Word({onPhraseClick, greekWords, englishWords, isPhrase, containsSubWords, handleClick} : WordProps) {
+  
+    const wordRef = useRef(null);
+  
+  
+    if(!greekWords)
+    {
+      return(
+        <>
+          <span>
+            {englishWords}
+          </span>
+          <span> </span>
+        </>
+      )
+    }
+    else if(isPhrase)
```

Comment:
Since the previous if statement contains a "return" statement, the "else" here is unnecessary.  I recommend removing the "else"s on this page to make it easier to read and maintain.